### PR TITLE
NLP-116 Understand 'X nights' as duration

### PR DESCRIPTION
**Pull request by jonaspok**

### CODE REVIEW

**Review by km1414 — APPROVED** [review]

**Comment by pranas** [comment]
I think we should keep the previous Docker setup: it makes builds and tests faster, you will appreciate it if you spend more time running builds on this. Also, adding a test in a duckling test suite would be nice so we get immediate feedback when working on this rather than having to run tests in a different project. Plus double check that failing test cases do not change before and after the change.

**Comment by jonaspok** [comment]
> I think we should keep the previous Docker setup: it makes builds and tests faster, you will appreciate it if you spend more time running builds on this. Also, adding a test in a duckling test suite would be nice so we get immediate feedback when working on this rather than having to run tests in a different project. Plus double check that failing test cases do not change before and after the change.

Reverted back to previous docker image version with logging disabled as this prevents growing container in size. Also checked that code change does not introduce any new test failures. However I was not able to find an easy way to integrate 'X nights' test to duckling directly. But may be it is not a big problem as we will have this test implemented on NLU side as a part of CI?

File: Makefile [hunk 1]
```diff
@@ -1,16 +1,22 @@
 .PHONY: builder
 
-image:
-	DOCKER_BUILDKIT=1 docker build -t duckling:latest .
+IMAGE_NAME=us.gcr.io/eddy-travels-global/duckling:5
```
Comment (pranas) [inline Makefile]: having hard-coded 5 here can lead to issues, maybe leave out tagging out of Makefile

File: stack.yaml.lock [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+# For more information, please see the documentation at:
+#   https://docs.haskellstack.org/en/stable/lock_files
+
+packages: []
+snapshots:
+- completed:
+    size: 535263
+    url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/lts/9/10.yaml
+    sha256: ba6ba39d048b5a2054142209f00e7e456e30081c03ea72b6135983633b234535
+  original: lts-9.10
```
Comment (pranas) [inline stack.yaml.lock]: Also, is this necessary? Seems like something that was produced on a local run

Comment (jonaspok) [reply stack.yaml.lock]: > Also, is this necessary? Seems like something that was produced on a local run

Good point, removed.

File: Makefile (hunk 1 above)
Comment (jonaspok) [reply Makefile]: > having hard-coded 5 here can lead to issues, maybe leave out tagging out of Makefile

Changed back to duckling:latest

**Review by pranas — APPROVED** [review]

**Pull request merged**